src/AngleSharp.Diffing.Tests/DiffingTestFixture.cs

```diff
@@ -11,7 +11,11 @@ public class DiffingTestFixture
 
         public DiffingTestFixture()
         {
-            var config = Configuration.Default.WithCss();
+            // Create a custom config with a parser to allow access to the source reference from the AST.
+            var config = Configuration.Default
+                .WithCss()
+                .With<IHtmlParser>(ctx => new HtmlParser(new HtmlParserOptions { IsKeepingSourceReferences = true, IsScripting = ctx?.IsScripting() ?? false }, ctx));
+
```

Comment:
So this needs to be in-place to make the compares tests actually work, right?

Reply:
Yes, otherwise SourceReference is always null.